src/client/datascience/notebook/intellisense/jupyterCompletionProvider.ts

```diff
@@ -61,22 +76,29 @@ export class JupyterCompletionProvider implements CompletionItemProvider {
         const timeout =
```

Comment:
Shouldn't we exclude pylance completions if we're dealing with a Julia, or R or other cell.

Reply:
They wouldn't come back anyway.

Reply:
The languageClient would skip them as they don't match it's document filter.

Reply:
And since this completion provider is only registered for python cells, it doesn't get called in that scenario anyway.